executor/delete.go

```diff
@@ -240,6 +246,7 @@ func (e *DeleteExec) removeRow(ctx sessionctx.Context, t table.Table, h kv.Handl
 
 // Close implements the Executor Close interface.
 func (e *DeleteExec) Close() error {
+	e.memTracker.ReplaceBytesUsed(0)
```

Comment:
We should replacebytesUsed to 0 after all children is closed.. 
If the children consume a negative number, the memory usgae will overflow

Reply:
Did you mean defer it?
Actually, I never imagined there is a case of negative memory footprint and wondered why it's `int64` rather than `uint64`.